2-dom-manipulation/form.js

```diff
@@ -0,0 +1,71 @@
+const form = document.querySelector("form");
+const nameField = document.querySelector("#name");
+const lastNameField = document.querySelector("#lastName");
+const emailField = document.querySelector("#email");
+const phoneNumberField = document.querySelector("#phone");
+const radioInputs = document.querySelectorAll("#estrato");
+const inputs = document.querySelectorAll("input");
+
+const expressions = {
+  name: /^([a-zA-Z]+\s?){2}?$/,
+  email: /^[a-zA-Z0-9._-]+@[a-zA-z0-9._-]+\.[a-zA-Z]+$/,
+  phoneNumber:
+    /^(\+\d{2})?[\s\-]?(\d{3})[\s\-]?(\d{2,3})[\s\-]?(\d{2})[\s\-]?(\d{2})$/,
+};
+
+function validateForm(event) {
+  switch (event.target.name) {
+    case "name":
+      if (expressions.name.test(event.target.value)) {
+        nameField.classList.add("correctField");
+        nameField.classList.remove("errorField");
```

Comment:
Since you're using the same strings multiple times, it makes sense to save them in consts.
```suggestion
        nameField.classList.add(CORRECT_FIELD);
        nameField.classList.remove(ERROR_FIELD);
```

Reply:
Ready, visible changes on the next commit.